lib/FocusTrap.js

```diff
@@ -1,31 +1,91 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import ReactDOM from 'react-dom';
+import {addFocusListener, addBlurListener} from './focusListeners';
 
 class FocusTrap extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.focusListener;
+    this.blurListener;
+    this.willWrap = !this.props.noWrapper;
```

Comment:
Should `willWrap` not be a part of the component's `state` to trigger an immediate re-render in the event that it is set to `false` after the component has mounted? Otherwise, if I am interpreting the code and component lifecycle correctly, the component will need to wait for some change in props to re-render the component.

Reply:
For instances where the component attempts to `findDOMNode()` but fails - ok, will do.